.clang-format

```diff
@@ -78,7 +78,7 @@ PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 60
 PointerAlignment: Left
 ReflowComments:  true
-SortIncludes:    true
```

Comment:
Does this correspond to our coding conventions?

Reply:
The problem is that headers of highfive need to be in a fixed order. Several peoples tried to fix it but unsucessfully iirc.

Reply:
Sorry, meant the whole file 😅

Reply:
This file was already there...
Should i copy one from hpc-coding-convention?

Reply:
Formatting config should the closest possible to hpc-coding-conventions. Of course, any rule can be overridden, like the `SortIncludes`

Reply:
> This file was already there... Should i copy one from hpc-coding-convention?

Yes please. Or re-add the submodule and link to the corresponding file.